app/assets/stylesheets/screens/_papers.scss

```diff
@@ -199,6 +199,29 @@ html.control-bar-sub-nav-active #paper-body .oo-ui-toolbar {
   overflow: hidden;
 }
 
+.recent-activity-link {
+  position: relative;
```

Comment:
Properties should be ordered color, cursor, font-size, height, line-height, margin-top, padding, position